### ISSUE TITLE
Translations - WirePayments module - missing entries

### ISSUE THREAD

**Issue by jaguart** [PrestaShop/PrestaShop#25041]
#### Describe the bug

I need to override a translation in the WirePayments Module.

WirePayments includes the translatable phrase: 'Your order will be sent as soon as we receive payment.'

In Admin -> Translation, when I select ps_wirepayment module no phrases are listed for translation.

If I add a row into the ``translation`` table in the database, and clear the cache, the phrase is correctly translated.

#### Expected behaviour

I expect that when I select module ps_wirepayment the translatable phrases for this module would be listed, and available for customisation.

#### Steps to Reproduce

Steps to reproduce the behaviour:

In Admin/BO 
  1. Improve -> International -> Translations -> Modify Translations:
    * Type of Translation -> Installed modules translations 
    * Select your module:  -> Prestashop Checkout
    * Language -> English
    * Modify
      * scroll to the end
      * Select your module: ps_wirepayment
      * Modify translations
        *  no phrases are listed for translation

**Screenshots**

Screenshot attached

#### Additional information

* PrestaShop version: 1.7.7.4
* PHP version: 7.3

![presta-001](https://user-images.githubusercontent.com/29786965/122650830-e3782480-d12c-11eb-83ae-3106c14d3a9c.jpg)

**Comment by hibatallahAouadni** [comment]
Hello @jaguart 

If you want to change the translation of **ps_wirepayment** module, all you need to do is  to select **Bank transfer** as module:

![image](https://user-images.githubusercontent.com/23308427/122743260-21895b80-d287-11eb-920c-c3b736801e70.png)

Please check and feedback.
Thanks!

**Comment by jaguart** [comment]
Yes - that worked, thank you, I can see the term I inserted in the database is there.

The Translation system makes it so difficult to find the module - I had spent several hours looking through all the options for all the modules. I never guessed that **ps_wirepayment** is called **Bank Transfer** in the Translation system, and then **ModulesWirepaymentShop** in the translation table.

It would be so much easier if there was a search for the term being translated, rather than requiring the users to manually read through every module and every term available for translation.

Thanks again.
